docs/build/html/_sources/changelog.rst.txt

```diff
@@ -14,7 +34,7 @@ Minor new features:
 * New ``ECR`` OpType.
 * New ``SynthesiseOQC`` pass.
 * New ``RebaseOQC`` pass.
-
+  
```

Comment:
Is this a wanted change?

Reply:
No, but it doesn't affect the formatting in the end. Will remove the whitespace next time I'm editing the changelog.